micro_manager/micro_manager.py

```diff
@@ -156,14 +164,17 @@ def solve(self) -> None:
         - Read data from preCICE, solve micro simulations, and write data to preCICE
         - If adaptivity is on, compute micro simulations adaptively.
         """
-        t, n = 0, 0
+        self._t = self._n = 0
```

Comment:
Can this potentially lead to the same reference, which leads to one variable being automatically modified when the other one is modified?